### DISCUSSION TITLE
Is password-protection a good idea with BLE-enabled devices?

### DISCUSSION THREAD

**Discussion by builder555** [Ralim/IronOS#1561] [General]
At the moment I can connect to any Pinecil V2 that's within range of my computer and make any changes. Should BLE interface require some sort of authentication or implement some other [security feature](https://antonioupan6.medium.com/bluetooth-low-energy-ble-security-in-a-nutshell-5d20164cceb2)?
I have no idea how prevalent IronOS is on BLE-enabled devices, so this may be completely irrelevant.

**Comment by Ralim** [comment]
This only will affect PinecilV2.
Security is my main reason for having the enable toggle.
Security via encryption in the BLE layer is completely broken.
In the past its been easier and more secure to implement a pin code lock than trust the BLE mess (especially getting support across phones).

Not against this being done, just more work ^_^
